index.test.js

```diff
@@ -0,0 +1,70 @@
+const { withSVG, withRasterImages, withPlayback } = require('./index.js');
+
+const options = {
+    dev: true,
+    isServer: true,
+};
+
+let config = {
+    module: {
+        rules: [],
+    },
+};
+
+const mockFunction = jest.fn();
+
+afterEach(() => {
+    config = {
+        module: {
+            rules: [],
+        },
+    };
+
+    jest.clearAllMocks();
+});
+
+it('should return a single rule', () => {
```

Comment:
One thing that also caught my eye is that this test is testing all plugins for one single rule. This is currently true, but in the future we might have a plugin that adds more than one rule.

A better way to do this is by describing our tests by context. In this case, by plugin. Example:
```js
describe('playback', () => {
    it('should return a single rule', () => {
      ...
    });
});

describe('svg', () => {
    it('should return two rules', () => {
      ...
    });
});
```

Another way is to create test files for each plugin. If we opt to have all plugins inside a folder like `plugins` or `lib` this makes it easier. Something like:
```
lib
├─ playback
       ├─ index.js
       ├─ test.js
... 
```
This way we create tests specific for each context.

Reply:
Yes, thanks, i'll follow the suggested structure. It also has immediate benefits, because the svg plugins already has specific tests that don't need to pollute a generalized tests file.

Reply:
I also agree to co-locate tests, but the folder seems unnecessary.

Why don’t we do it like this:

```
lib/
  playback.js
  playback.test.js
  raster.js
  raster.test.js
  ...
```
 Because the project is small, and won’t grow much, adding deep folders yields no benefits. Actually, it would be harder for people to navigate in the project.

Reply:
If we can presume that this project doesn't need to take significant scaling into consideration i think this is a good balance between cleanliness and avoiding unnecessary nesting. 

On another topic, do you have a strong opinion on `lib` versus `plugins`? I understand the former is a standard, but i think naming it plugins would keep up with the expectations given the title of the repository, and communicate more strongly the structure of the project.

Reply:
Not a strong opinion, but I usually follow the community convention of the lib folder

Reply:
I'll change it to `lib` to keep up with other Moxy repositories then, as i think the team will likely be the most common developer going through this repository.